docs: add mainnet to `README.md` (#192)
kroma-network/kroma: README.md

<div align="center">
  <br />
  <br />
  <a href="https://kroma.network"><img alt="Kroma" src="https://github.com/kroma-network/kroma-brand-kit/raw/main/assets/images/signature/Kroma-signature.svg" width=500></a>
  <br />
  <h3><a href="https://kroma.network">Kroma</a> aims to be a New Universal ZK Rollup on Ethereum.</h3>
  <br />
</div>

## About Kroma

> Kroma aims to develop a universal ZK Rollup based on the Optimism Bedrock architecture. Currently, Kroma operates as an

Reviewer: ```suggestion
Kroma is the first stage 2 universal Layer 2 on Ethereum. Kroma has been developed based on Optimism Bedrock architecture, and aims to a universal ZK Rollup. Currently, Kroma operates as an
```

Optimistic Rollup with ZK fault proofs, utilizing a zkEVM based on Scroll. The goal of Kroma is to eventually transition
to a ZK Rollup once the generation of ZK proofs becomes more cost-efficient and faster.

Kroma offers 10x lower transactions fees compared to the Ethereum mainnet, with native Ethereum security and Ethereum
equivalence. Our roadmap to transition from Optimistic rollup to ZK rollup sets Kroma apart from other L2 networks.

> Additionally, everyone in Kroma has the opportunity to participate as a validator. You can read our
> article '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
> to learn more.

Reviewer: ```suggestion
Additionally, Kroma is advancing towards decentralization. As an initial step for this, Kroma has introduced a permissionless validator and challenge system. You can read our
article titled '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
to learn more.
```

Kroma is live on the mainnet on 09/05/23, and we're working on optimizing ZK Proofs to transition to ZK Rollup as part

> of our roadmap. also Kroma is the first rollup protocol to achieve **'Stage 2'**.

Reviewer 2: ```suggestion
of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.
```

> of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.

Reviewer: I think this is not necessary here. But let's add the word `the first universal state 2 rollup` at the start of this description.

For more detailed information about Kroma, check [Kroma docs](https://docs.kroma.network) (or

> [Markdown docs source](./specs)).

Reviewer: ```suggestion
[specs](./specs)).
```

## Links

- Official Website: [Kroma](https://kroma.network)
- Documentation: [Kroma Docs](https://docs.kroma.network)
- Blog: [Kroma Medium](https://medium.com/@kroma-network)

> - Discord: [Kroma](https://discord.gg/kroma)

Reviewer: ```suggestion
- Discord: [Kroma discord](https://discord.gg/kroma)
```

> - X: [Kroma](https://twitter.com/kroma_network)

Reviewer: ```suggestion
- X: [Kroma X](https://twitter.com/kroma_network)
```
